src/sinks/prometheus/remote_write.rs

```diff
@@ -147,6 +147,10 @@ impl SinkConfig for RemoteWriteConfig {
     fn sink_type(&self) -> &'static str {
         "prometheus_remote_write"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
This is HTTP-based, no?

Reply:
It is, but we are handling the HTTP layer directly in the sink instead of managing it with one of the `HttpSink` wrappers, and AFAICT we're just handing up the response to tower and not using it to drive acknowledgements.

Reply:
Is my understanding correct that this sink _could_ support acknowledgements if it was using our `Service` rather than `tower::Service` directly?

Reply:
Correct.